apps/dynatrace/dynatrace-crds/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
How are the Dynatrace-crds being applied?

Reply:
CRD path added to base.